pkg/tsdb/cloud-monitoring/cloudmonitoring.go

```diff
@@ -65,11 +59,10 @@ const (
 	perSeriesAlignerDefault   = "ALIGN_MEAN"
 )
 
-func ProvideService(httpClientProvider httpclient.Provider, tracer tracing.Tracer) *Service {
```

Comment:
Instead of removing this I think we should use the tracer from the SDK? I think that's applicable? cc @andresmgot

Reply:
I think that the tracer is still used: [link](https://github.com/grafana/grafana/pull/80181/files#diff-e0d8854117661da605b3d4b545db968683557373389b57e2a8143ce13ef487e3R128). We probably don't need to store it in the service (I'm no tracer expert :D)

Reply:
That's correct, `backend.DefaultTracer()` is the tracer provided by the SDK and the dependency from the tracer in `pkg/infra/tracing` can be removed